rcore/flash.c

```diff
@@ -117,8 +117,16 @@ struct page_hdr {
 /* flag values are *cleared* to mean "true" */
 #define FLASHFLAG(val, flag) (((val) & (flag)) == 0)
 
-static void _fs_read_page_hdr(int pg, struct page_hdr *p) {
-    flash_read_bytes(REGION_FS_START + pg * REGION_FS_PAGE_SIZE, (uint8_t *)p, sizeof(struct page_hdr));
+#define MAX_FILENAME_LEN 64
```

Comment:
Where'd this constant come from?  Is that actually the maximum?

Reply:
just my assumption, will comment and add check in `init`